### ISSUE TITLE
Login broken after Upgrade to 4.0.4 from 3.x on Android

### ISSUE THREAD

**Issue by fbuchmeier-abi** [RocketChat/Rocket.Chat.ReactNative#1380]
- Your Rocket.Chat Experimental app version: 4.0.4
- Your Rocket.Chat server version: 2.2.0
- Device (or Simulator) you're running with: Samsung Galaxy S8 Android 9

Hi,

after upgrading the mobile app to 4.0.4, login at our server (version 2.2.0) is completely broken.

We are using the following environment:
```
Tag | 2.2.0
OS Type | Linux
OS Release | 4.15.0-66-generic
Node Version | v8.11.4
Mongo Version | 3.6.12
Mongo Storage Engine | wiredTiger
```
With the latest mobile application running on a Samsung Galaxy S8 with Android 9 and the patch level from Oct-2019.

According to the mobile applications, login credentials (using the Atlassian Crowd connector) are rejected.
When looking at the HTTP response code (401) in our reverse proxy (NGINX) this makes sense:
```
172.23.16.231 - - [13/Nov/2019:13:09:10 +0000] "POST /api/v1/login HTTP/1.1" 401 115 "-" "RC-RN Mobile/4.0.4 (build: 2090; os: android 9)"
```
So I increased the log level in our Rocket.Chat server instance to `Errors, Information and Debug` enabled all the tracing options and looked at the output, which is a single line upon login:
```
server.js:204 API ➔ debug POST: /api/v1/login
```

Log in with a Browser (Chromium) and the same user & password works fine without any problems.

After downgrading the mobile application to 3.5.1 which can be found for example in the F-Droid marketplace https://f-droid.org/en/packages/chat.rocket.android/ everything works as expected again.

How can we debug the issue further?

Thanks and regards,
Florian.

**Comment by diegolmello** [comment]
@fbuchmeier-abi Can you reach me at https://open.rocket.chat? I'm `@diego.mello`.
Thanks.

**Comment by diegolmello** [comment]
Attempted to fix on #1381.
If everything is fine, I'll release 4.0.5 later today.

**Comment by diegolmello** [comment]
@fbuchmeier-abi Fix released as 4.0.5 on Google Play.
Thanks for your help debugging!
